app/renderer/js/utils/reconnect-util.js

```diff
@@ -0,0 +1,52 @@
+const isOnline = require('is-online');
+
+class ReconnectUtil {
+	constructor(serverManagerView) {
+		this.serverManagerView = serverManagerView;
+		this.alreadyReloaded = false;
+	}
+
+	clearState() {
+		this.alreadyReloaded = false;
+	}
+
+	pollInternetAndReload() {
+		const pollInterval = setInterval(() => {
+			this._checkAndReload()
+			.then(status => {
+				if (status) {
+					this.alreadyReloaded = true;
+					clearInterval(pollInterval);
+				}
+			});
+		}, 1500);
+	}
+
+	_checkAndReload() {
+		return new Promise(resolve => {
+			if (!this.alreadyReloaded) { // eslint-disable-line no-negated-condition
+				isOnline()
+				.then(online => {
+					if (online) {
+						if (!this.alreadyReloaded) {
+							this.serverManagerView.reloadView();
+						}
+						console.log("You're back online.");
```

Comment:
console.log('You're back online.');

Reply:
@cPhost this.

Reply:
Ah. i forgot to `git add` this :)

Reply:
I though if you had an `'` inside a string that needs to escaped we can use double quotes.